build/pkgs/sirocco/spkg-configure.m4

```diff
@@ -0,0 +1,10 @@
+SAGE_SPKG_CONFIGURE([sirocco], [
+ SAGE_SPKG_DEPCHECK([mpfr], [
+  PKG_CHECK_MODULES([libsirocco], [libsirocco >= 2.1.0], [
```

Comment:
This will reject all the old versions of sirocco that not yet have a pkg config file, right? Do older versions work in principle?

Reply:
oh, right. I will fix it, sorry

Reply:
[4932425](https://github.com/sagemath/sage/pull/40555/commits/4932425cfb1f6bf77c311c33629fcd3474a6a9ce) - fixed